src/renderer/components/Sidebar.vue

```diff
@@ -1,70 +1,73 @@
-        <span class="icon">
-          <aph-icon name="dashboard"></aph-icon>
-        </span>
-        <span class="label">{{$t('dashboard')}}</span>
-      </router-link>
-      <router-link v-if="currentNetwork.net !== 'MainNet'" to="/authenticated/dex">
-        <span class="icon">
-          <aph-icon name="dex"></aph-icon>
-        </span>
-        <span class="label">{{$t('tradeDEX')}}</span>
-      </router-link>
-      <router-link v-else to="/authenticated/buy-aph">
-        <span class="icon">
-          <aph-icon name="dex"></aph-icon>
-        </span>
-        <span class="label">{{$t('buyAph')}}</span>
-      </router-link>
-      <router-link to="/authenticated/assets">
-        <span class="icon">
-          <aph-icon name="wallet"></aph-icon>
-        </span>
-        <span class="label">{{$t('assets')}}</span>
-      </router-link>
-      <router-link to="/authenticated/history">
-        <span class="icon">
-          <aph-icon name="history"></aph-icon>
-        </span>
-        <span class="label">{{$t('history')}}</span>
-      </router-link>
-      <router-link to="/authenticated/token-sale">
-        <span class="icon">
-          <aph-icon name="ico"></aph-icon>
-        </span>
-        <span class="label">{{$t('joinIco')}}</span>
-      </router-link>
-      <router-link to="/authenticated/settings">
-        <span class="icon">
-          <aph-icon name="settings"></aph-icon>
-        </span>
-        <span class="label">{{$t('settings')}}</span>
-      </router-link>
-    </div>
-    <div class="logout-wrapper" @click.prevent="logout">
-      <aph-icon name="logout"></aph-icon>
-    </div>
-    <div class="footer link-list">
-      <div class="network-status">
-        <div class="block">
-          <span class="network">{{ currentNetwork ? currentNetwork.net : 0 }} {{$t('block')}}</span>
-          <span class="index">{{ currentNetwork && currentNetwork.bestBlock ? currentNetwork.bestBlock.index : 0}}</span>
-        </div>
-        <div class="last-update">
-          <div v-if="showNetworkError" class="network-error">{{$t('unableToReachNetwork')}}</div>
-          <div v-else>
-            <aph-timestamp-from-now :timestamp="lastReceivedBlock"></aph-timestamp-from-now>
+  <section id="sidebar" :class="{'collapsed': toggleable && collapsed, 'expanded': toggleable && !collapsed}">
+    <aph-icon v-if="toggleable" @click.native="collapsed = !collapsed" class="toggle" :name="collapsed ? 'double-arrow-right' : 'double-arrow-left'"></aph-icon>
+    <template v-if="!toggleable || (toggleable && !collapsed)">
+      <div class="header">
+        <aph-icon name="logo-mark"></aph-icon>
+      </div>
+      <div class="menu link-list">
+        <router-link :to="sendInProgress ? '/authenticated/dashboard/confirming': '/authenticated/dashboard'">
+          <span class="icon">
+            <aph-icon name="dashboard"></aph-icon>
+          </span>
+          <span class="label">{{$t('dashboard')}}</span>
```

Comment:
Spaces between curly braces and expression.

Reply:
done